variables.tf

```diff
@@ -129,3 +129,9 @@ variable "emr_abac_valid_tags" {
   description = "A map of valid tags for maintaining EMR resources when using ABAC IAM Policies with Tag Conditions. Make sure your tamr-config.yml file specifies tags key values. Refer to tamr-config module examples for more info."
   default     = {}
 }
+
+variable "private_ips" {
+  type        = list(string)
+  default     = []
```

Comment:
Assuming setting this to an empty list is equivalent to the previous behavior. Has that been tested?

Reply:
Ran a couple more tests and indeed it looks like there is an issue with passing this default to the network interface. While relying on the default did result in it choosing an IP on it's own, later running some changes resulted in the following error
```
Error: Failure to unassign Private IPs: InvalidParameterValue: Value (10.1.0.208) for parameter privateIpAddress is invalid. The primary IP address of an interface cannot be unassigned.
        status code: 400, request id: c5fe7105-51fb-4a8c-9f2f-7dfddb906b1e

  on .terraform/modules/tamr-vm/modules/aws-ec2-instance/main.tf line 25, in resource "aws_network_interface" "tamr-instance-network":
  25: resource "aws_network_interface" "tamr-instance-network" {
```

The error goes away if then specify explicitly the IP that was chosen during the initial deployment.  So it seems need to treat the default another way when passing to the network interface. Is there a preferred way to do this? Perhaps using [nullable arguments](https://www.terraform.io/docs/language/expressions/types.html#null)?

Reply:
yeah, setting `null` might be equivalent to what was happening prior to this change

Reply:
Updated defaults to `null` and confirmed works as previously